.clj-kondo/config.edn

```diff
@@ -298,7 +298,10 @@
     metabase.audit                 #{metabase.audit}
     metabase.bootstrap             #{metabase.bootstrap}
     metabase.cmd                   #{}                                                              ; there are no namespaces here since you shouldn't be using this in any other module.
-    metabase.channel               #{metabase.channel.core}
+    metabase.channel               #{metabase.channel.core
```

Comment:
in the next PR I'll rework the APIs so that only metabase.channel.core is the public namespace.